playbooks/roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
I was merely asking about current customers that are running inside docker containers.